### PR TITLE
chore(ci): add CI based on sonata projects one

### PR DESCRIPTION
**Pull request by bmaziere**
Travis integration has been activated in settings

### CODE REVIEW

File: .php_cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<?php
+$header = <<<'HEADER'
+
+This file is part of the Ekino project.
```
Comment (Bladrak) [inline .php_cs]: Sonata for Ekino project?

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2018 Ekino PHP Team
```
Comment (Bladrak) [inline LICENSE]: Ekino seems sufficient, or we need to update the header in php-cs for consistency

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+	@grep -E '^[a-zA-Z_-]+:.*?##.*$$' $(MAKEFILE_LIST) | sort | awk '{split($$0, a, ":"); printf "\033[36m%-30s\033[0m %-30s %s\n", a[1], a[2], a[3]}'
+
+cs-check: ## to show files that need to be fixed
+	php-cs-fixer fix --ansi --verbose --diff --dry-run
+
+cs-fix: ## to fix files that need to be fixed
+	php-cs-fixer fix --verbose
+
+lint: ## to lint php, yaml & composer
+	lint-composer lint-composer cs-check lint-yaml
```
Comment (Bladrak) [inline Makefile]: Shouldn't this be on the same line as ``lint:`` ?

File: phpunit.xml.dist [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+         convertErrorsToExceptions="true"
+         convertNoticesToExceptions="true"
+         convertWarningsToExceptions="true"
+         processIsolation="false"
+         stopOnFailure="false"
+         syntaxCheck="false"
+         bootstrap="tests/bootstrap.php"
+>
+    <testsuites>
+        <testsuite name="SonataHelpersBundle Test Suite">
```
Comment (Bladrak) [inline phpunit.xml.dist]: Not sure about the name of the test suite, maybe Sonata Ekino Test Suite? I don't see any bundle referenced here.

File: Makefile (hunk 3 above)
Comment (bmaziere) [reply Makefile]: added make call before lint-composer :+1: (i reserved same line for help description)

**Comment by Bladrak** [comment]
Seems great so far 👍

**Comment by bmaziere** [comment]
yeah :) need to find out why 
Fatal error: Uncaught Error: Class 'Symfony\Bridge\PhpUnit\DeprecationErrorHandler' not found in /home/travis/build/ekino/sonata/vendor/symfony/phpunit-bridge/bootstrap.php on line 39

that makes the build failed

**Comment by bmaziere** [comment]
think we are pretty good @bdejacobet @Wharenn @srascar @Bladrak @qsomazzi

File: .travis.yml [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+  - if [[ ! -z ${RELEVANT} ]];then exit 0; fi;
+  - if [ -x .travis/before_install_${TARGET}.sh ]; then .travis/before_install_${TARGET}.sh; fi;
+
+install:
+  - if [ -x .travis/install_${TARGET}.sh ]; then .travis/install_${TARGET}.sh; fi;
+
+before_script:
+  - if [ -x .travis/before_script_${TARGET}.sh ]; then .travis/before_script_${TARGET}.sh; fi;
+
+script: make $TARGET
```
Comment (nanofelis) [inline .travis.yml]: reminder for adding the lint task to the process

Comment (bmaziere) [reply .travis.yml]: done @nanofelis :) thx for the reminder

**Comment by Bladrak** [comment]
I'm guessing we have enough approvals, but what should be our policy on merging?

**Comment by bmaziere** [comment]
agree @Bladrak => https://github.com/ekino/sonata/issues/10

**Pull request merged**